pkg/cmd/cli/secrets/link_secret_to_obj.go

```diff
@@ -0,0 +1,190 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// LinkSecretRecommendedName `oc secrets link`
+	LinkSecretRecommendedName = "link"
+
+	// AddSecretRecommendedName Maintained for backwards-compatibility, but deprecated
+	AddSecretRecommendedName = "add"
+
+	// TODO: move to examples
+	linkSecretLong = `
+Link secrets to a ServiceAccount
```

Comment:
Should it be "service account" here for consistency since you use lower-case "secrets" in the same sentence and "service account" below?